### ISSUE TITLE
Pylance: Expression of type "ForeignKeyRelation[Unknown]" cannot be assigned to declared type

### ISSUE THREAD

**Issue by jleclanche** [tortoise/tortoise-orm#993]
**Describe the bug**
![image](https://user-images.githubusercontent.com/235410/143927355-d8d1d461-6236-4064-9d2f-f27e0f3b6121.png)

**To Reproduce**

Analyze the following with Pylance in vscode:

```py
class User(Model):
    pass

class Profile(Model):
   user: fields.ForeignKeyRelation[User] = fields.ForeignKeyField(
        "models.User", on_delete=fields.SET_NULL, null=True, blank=True
    )
```

**Expected behavior**
Mypy seems to think it's okay but Pylance doesn't. I can't be sure where the issue lies and I've already spent quite a bit of time investigating this to no avail.

**Comment by jleclanche** [comment]
Note this follows the documentation.

https://tortoise-orm.readthedocs.io/en/latest/models.html#improving-relational-type-hinting

**Comment by BobDotCom** [comment]
Fixed in aa3d51126065f352e21f7e1531b09547e54aee97 👍

**Comment by jleclanche** [comment]
Confirming fixed, thank you!
